tests/mozhttpd_base_test.py

```diff
@@ -6,52 +6,41 @@
 
 from __future__ import absolute_import, print_function, unicode_literals
 
+import logging
 import os
 import tempfile
 import unittest
 
 import mozfile
-import mozhttpd
-import mozlog
-
-from mozdownload.utils import urljoin
+from wptserve import server
 
 WDIR = 'data'
 HERE = os.path.dirname(os.path.abspath(__file__))
 
 
-@mozhttpd.handlers.json_response
-def resource_get(request, objid):
-    """Request handler for JSON responses."""
-    return (200, {'id': objid, 'query': request.query})
-
-
 class MozHttpdBaseTest(unittest.TestCase):
```

Comment:
In the future it might be good to get rid of the HTTP base class but make use of a decorator for those tests which need the server. Performance wise we might also want to start it once and not for each test.